jenkinsci/jenkins
# [JENKINS-73114] avoid conflicts with css classes from bootstrap

jenkins and bootstrap both have definitions for `alert` and `alert-warning`, `alert-info` and `alert-danger`. Bootstrap css definitions are included when e.g. the warning-ng plugin is installed and a job is configured to scan for warnings. The css classes from bootstrap are then chosen instead of the ones for Jenkins for things like the admin monitors. To make things unambiguous, add additional classes prefixed with `jenkins-` and make use of them in code. Keep the old definitions for backward compatibility with plugins.
Follow-up changes are needed in plugins (Mostly those that define admin monitors) and the design-library

After:
for a plugin message:
![image](https://github.com/jenkinsci/jenkins/assets/17767050/c076a5b8-a415-4930-a03e-90c16ba3daa8)

for a core message:
![image](https://github.com/jenkinsci/jenkins/assets/17767050/885b53dd-ebf6-46fa-8b09-fd1fee5e7309)

See [JENKINS-73114](https://issues.jenkins.io/browse/JENKINS-73114).

Originally reported as https://github.com/jenkinsci/customizable-header-plugin/issues/99


### Testing done

Manual testing
1. Create new Jenkins instance, it should have some executors for built-in, this will trigger the corresponding monitor
2. Visit page `/manage/` -> monitors are displayed properly
3. install warnings-ng plugin
4. create freestyle job and configure post build action to scan for e.g. stylellint.
5. run the job once
6. In the header click on the warning shield -> monitor is displayed properly (If there are monitors from plugins they might not be properly styled here)


### Proposed changelog entries

- Add new CSS classes to avoid conflicts with CSS classes from bootstrap.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
```

### Desired reviewers

@mention



Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
```
